android/jni/app-android.cpp

```diff
@@ -688,8 +688,9 @@ extern "C" void Java_org_ppsspp_ppsspp_NativeApp_init
 	EARLY_LOG("NativeApp.init() -- begin");
 	PROFILE_INIT();
 
-	std::lock_guard<std::mutex> guard(renderLock);  // Note: This is held for the rest of this function - intended?
+	std::lock_guard<std::mutex> guard(renderLock);  // Note: This is held for the rest of this function - intended? and even necessary?
```

Comment:
I would think so.  I think it should be on shutdown too.  It should not be possible to update `renderer_inited` to true, update `renderer_inited` to false, or render at the same time.  Those operations should all be sequenced.  Remember that there are annoying things happening with resize events and etc.

-[Unknown]

Reply:
Yeah, true. I'll throw in a lock in shutdown too.